src/stp/new_plays/Pass.cpp

```diff
@@ -42,36 +41,34 @@ void Pass::assignRoles() noexcept {
     flagMap.insert({"test_role_9", {closeToBallFlag}});
     flagMap.insert({"test_role_10", {closeToTheirGoalFlag}});
 
-    auto distribution = dealer.distribute(world->getWorld()->getUs(), flagMap);
-
-    stpInfos = std::unordered_map<std::string, StpInfo>{};
-    for (auto& role : roles) {
-        auto roleName{role->getName()};
-        if (distribution.find(roleName) != distribution.end()) {
-            auto robot = distribution.find(role->getName())->second;
-
-            stpInfos.emplace(roleName, StpInfo{});
-            stpInfos[roleName].setRobot(robot);
-        }
-    }
+    return flagMap;
 }
 
 void Pass::calculateInfoForRoles() noexcept {
     // Calculate most important positions to defend
     // You know you have n defenders, because the play assigned it that way
     auto enemyRobots = world->getWorld()->getThem();
-    auto defensivePositions = calculateDefensivePositions(2, world, enemyRobots);
+    int numberOfDefenders = 2;
```

Comment:
This can be const I think
```suggestion
    const int numberOfDefenders = 2;
```